spec/lib/diaspora/exporter_spec.rb

```diff
@@ -10,6 +17,14 @@
       user,
       %i[generic_user_data activity status_messages_flavours work_aspect]
     )
+
+    # verifications of data generation (protect us from possible false positives in case of poor data preset)
```

Comment:
I'm not sure about these "tests for tests". We want to test the code here, not the test. This `DataGenerator` is maybe an exceptions, so maybe we want to add some basic tests for the `DataGenerator` to test that it actually generates the data we want (but only the basics, not 100 test cases ;) ). But that should be somewhere else and not in the `exporter_spec`.